src/psyclone/domain/lfric/kern_call_arg_list.py

```diff
@@ -237,8 +237,32 @@ def stencil_unknown_extent(self, arg, var_accesses=None):
         '''
         # The extent is not specified in the metadata so pass the value in
         from psyclone.dynamo0p3 import DynStencils
-        name = DynStencils.dofmap_size_name(self._kern.root.symbol_table, arg)
-        self.append(name, var_accesses)
+        var_name = DynStencils.dofmap_size_name(self._kern.root.symbol_table,
+                                                arg)
+        name = "{0}({1})".format(var_name,
+                                     self._cell_ref_name(var_accesses))
```

Comment:
I suspect pylint will complain about the indentation of this line and 264.

Reply:
Fixed